components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabsToolbarFeature.kt

```diff
@@ -70,6 +81,31 @@ class CustomTabsToolbarFeature(
         }
     }
 
+    @VisibleForTesting
+    internal fun addActionButton(buttonConfig: CustomTabActionButtonConfig?) {
+        buttonConfig?.let { config ->
+            val button = Toolbar.ActionButton(
+                BitmapDrawable(context.resources, config.icon),
+                config.description
+            ) { config.pendingIntent.send() }
+
+            toolbar.addBrowserAction(button)
+        }
+    }
+
+    @VisibleForTesting
+    internal fun addShareButton(session: Session) {
+        val button = Toolbar.ActionButton(
+            ContextCompat.getDrawable(context, R.drawable.mozac_ic_share),
```

Comment:
Should we also want to include the `contentDescription` for accessibility purposes?